### PR TITLE
fix(uniswap-v2,substreams-helper): fix the build and bump dependencies

### LINKED ISSUE #5: uniswap-v2: build failure

### PR DESCRIPTION
**Pull request by 0xbe1**
close #5

### CODE REVIEW

**Comment by 0xbe1** [comment]
close https://github.com/messari/substreams/issues/5

**Comment by this-username-is-taken** [comment]
Let's follow the standard PR naming conventions from day 1: https://github.com/messari/subgraphs/blob/master/docs/Contributing.md#additional-identifiers

File: uniswap-v2/src/lib.rs [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
-use substreams_ethereum::{pb::eth::v1 as eth, Event as EventTrait};
-use substreams_helper::main;
+use substreams_ethereum::{pb::eth::v2 as eth, Event as EventTrait};
+use substreams_helper::{utils, erc20};
 
-use pb::erc721;
+// use pb::erc721;
 
-substreams_ethereum::init!();
+// substreams_ethereum::init!();
```
Comment (this-username-is-taken) [inline uniswap-v2/src/lib.rs:13]: is this no longer needed in 0.2.0?

File: uniswap-v2/src/lib.rs [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
 use pb::uniswap_v2;
 use abi::factory;
 use substreams::{log, proto, store};
-use substreams_ethereum::{pb::eth::v1 as eth, Event as EventTrait};
-use substreams_helper::main;
+use substreams_ethereum::{pb::eth::v2 as eth, Event as EventTrait};
+use substreams_helper::{utils, erc20};
 
-use pb::erc721;
+// use pb::erc721;
```
Comment (this-username-is-taken) [inline uniswap-v2/src/lib.rs:11]: feel free to remove this

File: uniswap-v2/substreams.yaml [hunk 3]
```diff
@@ -3,17 +3,17 @@ package:
...
-  erc721: ../erc721/substreams.yaml
+#imports:
+#  eth: https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg
+#  erc721: ../erc721/substreams.yaml
 
 protobuf:
   files:
     - uniswap_v2.proto
-    - erc721.proto
+#    - erc721.proto
```
Comment (this-username-is-taken) [inline uniswap-v2/substreams.yaml:13]: and the erc721-related code here

File: uniswap-v2/substreams.yaml [hunk 4]
```diff
@@ -3,17 +3,17 @@ package:
   name: "substreams_uniswap_v2"
   version: v0.1.0
 
-imports:
-  eth: https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg
-  erc721: ../erc721/substreams.yaml
+#imports:
+#  eth: https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg
```
Comment (this-username-is-taken) [inline uniswap-v2/substreams.yaml:7]: why comment out this import?

File: uniswap-v2/src/lib.rs (hunk 1 above)
Comment (0xbe1) [reply uniswap-v2/src/lib.rs:13]: yep

**Comment by 0xbe1** [comment]
> Let's follow the standard PR naming conventions from day 1: https://github.com/messari/subgraphs/blob/master/docs/Contributing.md#additional-identifiers

I find the convention `identifier(#semver-change); subgraph-name; description` not too aligned with today's substreams repo because it is not too useful to differentiate between patch, minor, and major atm.

so I propose just `identifier(modules): description` at the start.

@this-username-is-taken what do you think

**Comment by this-username-is-taken** [comment]
LGTM!

**Pull request merged**